lib/components/heading/test.js

```diff
@@ -0,0 +1,12 @@
+import Heading from './';
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+it('renders the heading with the respective heading element for h1-h6', () => {
+  [1, 2, 3, 4, 5, 6].forEach(function(v) {
+    const tree = renderer
+      .create(<Heading headingLevel={v} content={`${v} content`} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
snapshot testing here is a bit heavy-handed for what we want to test. that is, although the test _says_ that it's confirming the appropriate heading element it's really only making sure that the behavior doesn't change - right or wrong. the risk I'd like to avoid is failing a test when it's not wrong.

additionally, by running `forEach` inside of the test it seems like it can obscure the unit under test. how about using the `.each` test helper?

```js
test.each([1, 2, 3, 4, 5, 6])('renders h%d', level => {
  const wrapped = shallow(<Heading headingLevel={level} content="test" />);
  expect(wrapped.type()).toBe(`h${level}`);
});
```

Reply:
That would be a lot better. Fixed in 0414430.